### PR TITLE
control for 3.5" lcd screen

### PR DESCRIPTION
**Pull request by knilios**
The code has finally finalized.

**Added stuffs:** 
 1. Added the ability to display score on the 3.5" lcd screen.
 2. Fix bugs.

### CODE REVIEW

**Comment by knilios** [comment]
The code has finally finalized.

**Added stuffs:** 
 1. Added the ability to display score on the 3.5" lcd screen.
 2. Fix bugs.

**Review by NETtoSan — COMMENTED** [review]
the code works properly. it would be better if you describe the code's function. and not printing serial all the time it outputs.

**Review by NETtoSan — CHANGES_REQUESTED** [review]
pls dont serial.println every value it ommits. its messy

File: satitsport/scoreboard/master.ino [hunk 1]
```diff
@@ -91,7 +122,7 @@ void loop() {
       scoreTeam1 = scoreTeam1 + 1;
       myData.scoreA = scoreTeam1;
       myData.scoreB = scoreTeam2;
-    } else() {
```
Comment (NETtoSan) [inline satitsport/scoreboard/master.ino:94]: no wonder why it cant compile XDDD

**Review by NETtoSan — COMMENTED** [review]
this one works

**Review by NETtoSan — APPROVED** [review]

**Pull request merged**